### DISCUSSION TITLE
how to run plotman?

### DISCUSSION THREAD

**Discussion by ghost** [ericaltendorf/plotman#668] [General]
hi, really, how to run plotman?
using comand plotman nothin do, if i run plotman interactive, i have error:
![image](https://user-images.githubusercontent.com/20506433/119684409-26f0b300-be4d-11eb-944e-a201daa63bc1.png)
i saw that error is "No such file or directory: 'chia'" but how to fix this, i don't know. in plotman config, i don't have folder with this name, only my username is "chia"

**Comment by pspardon** [comment]
I think you need to activate your chia environment first before running `plotman interactive`

cd to your chia-blockchain directory and run `. ./activate`
